client/src/components/Modal/Modal.css

```diff
@@ -0,0 +1,83 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
```

Comment:
The font is provided globally, so you don't need to import it - just use the CSS variables